packages/core/discovery.config.default.js

```diff
@@ -6,6 +6,30 @@ module.exports = {
     author: 'Store Framework',
     name: 'FastStore',
     publisherId: '',
+    organization: {
```

Comment:
question: I'm wondering, since it's already configurable in the hCMS, why add it also on the discovery.config? I understand there are already some SEO config on this file and on hCMS, but I started to wonder about it now because it's a lot of data and we are about to add more SEO data in upcoming PRs. So I'd like to hear what you thought about it, and if I'm missing something

Reply:
Nice question! I just added it to follow the pattern, keeping it in both places and allowing customization via hCMS and code. However, I feel like we're adding more and more attributes. Maybe it would be better to create a separate file just for handling SEO configuration, or we could keep these "extra" settings in hCMS instead.

Reply:
Yeah!! Could you open a thread on our channel so everyone can discuss those approaches? 🙏

Reply:
We've discussed as a team, and decided to keep particular page's SEO configurations only in the hCMS.